manifests/test_templates.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: batch/v1
```

Comment:
have you run this manually with `kubectl create`? just to make sure this works without the added complexity of prodvana

Reply:
Yes. But probably want to use my own namespace instead of ansible.